CMakeLists.txt

```diff
@@ -51,9 +51,10 @@ include(${CMAKE_CURRENT_SOURCE_DIR}/cmake/Config.cmake)
 include(GNUInstallDirs)
 
 # setup version numbers
-set(VERSION_MAJOR 2)
-set(VERSION_MINOR 5)
-set(VERSION_PATCH 1)
+set(VERSION_MAJOR 3)
+set(VERSION_MINOR 0)
+set(VERSION_PATCH 0)
+set(VERSION_UNRELEASED ON)
```

Comment:
This might lead to writing double negations like `if(NOT VERSION_UNRELEASED)`.

Reply:
Good point, maybe `VERSION_IS_RELEASE` is better. 
The "is" also makes clear it's a boolean flag and not a version, as otherwise it follows the same naming as the 3 versions.

Reply:
Like this? 🙂